src/utils.ts

```diff
@@ -4,3 +4,36 @@ import { deepEqual } from "fast-equals"
 export function unique<T extends Record<string, any>>(array: T[]) {
   return array.filter((elm2, index) => array.findIndex((elm1) => deepEqual(elm1, elm2)) === index)
 }
+
+export function getIcon(iconType: string | undefined, kindType: string | undefined) {
+  // LSP specification: https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_documentSymbol
```

Comment:
Moved functions referenced from both the outline and call hierarchy to utils.ts.